google/resumable_media/requests/download.py

```diff
@@ -456,21 +520,21 @@ def _parse_md5_header(header_value, response):
 class _DoNothingHash(object):
     """Do-nothing hash object.
 
-    Intended as a stand-in for ``hashlib.md5`` in cases where it
-    isn't necessary to compute the hash.
+    Intended as a stand-in for ``hashlib.md5`` or another checksum in cases
```

Comment:
Suggestion:

`For md5 and crc32c when it isn't necessary to compute the hash.`

Reply:
I'd prefer to be specific about the classes it's emulating so I'll update this to be specific both about this class and that crc32c is accepted.